api/jsonapi_test.go

```diff
@@ -458,12 +462,13 @@ func TestCryptoListingsIllegalFields(t *testing.T) {
 	listing = factory.NewCryptoListing("crypto")
 	listing.Coupons = physicalListing.Coupons
 	runTest(listing, core.ErrCryptocurrencyListingIllegalField("coupons"))
+
 }
 
 func TestMarketRatePrice(t *testing.T) {
 	listing := factory.NewListing("listing")
 	listing.Metadata.Format = pb.Listing_Metadata_MARKET_PRICE
-	listing.Item.Price = 1
+	listing.Item.Price = &pb.CurrencyValue{Currency: &pb.CurrencyDefinition{Code: "BTC", Divisibility: 8}, Value: "1"}
```

Comment:
You can use `repo.NewCurrencyValue("1", btcDef)` with the `CurrencyDefinition` you looked up above to properly generate valid objects.